openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,38 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
```

Comment:
Can you do `if 'resources' not in data: return` here? So we avoid one extra level of indentation